Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,11 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const workHours = tasks.map(workHour => workHour['duration'] / 60);
+const workHoursMoreThan2 = workHours.filter(workHoursMoreT2 => workHoursMoreT2 > 2);
+let sum;
+function add(sum, elem) {
+  return sum + elem;
+}
+let totalIncome = workHoursMoreThan2.map(hour => hour * 10).reduce(add, 0);
```

Comment:
ESLint indicates that `totalIncome` can be a `const`. You should prefer `const` over `let` if you do not intend to reassign a variable. This helps to prevent reassigning this variable by mistake.

I assume that the number 10 in this statement means the rate that Maartje makes per hour. A number like this is sometimes called a _'magic number'_, because it is not immediately clear what it represents. It is better to create a constant that describes this magic value, e.g.:

```js
const hourlyRate = 10;
```